solidity/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
-build/
 .python-version
+build/contracts/*
+!build/contracts/Oracle.sol
```

Comment:
Only included Oracle.sol, as that's the only one we *need* right now?

Reply:
Ok. They will only be included in the binary if you run `truffle build` beforehand. Still not ideal though, and I'm open to other possible solutions.

Reply:
Good point.